src/database/migrations/20240501004030-create-size.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+'use strict';
+
+const sequelize = require('sequelize');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('sizes', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
```

Comment:
that's perfect model, however, I'd like to remind everyone, when declaring a primary key, you don't need to put a unique constraint cause primary key can never be duplicates